.travis.yml

```diff
@@ -8,62 +8,71 @@ jobs:
       os: linux
       name: "Compile & Formatting Check"
       env:
+        - CI="1"
```

Comment:
@nkohen thought this was setup for the whole environment? Can't we configure these to be matrix wide?

Reply:
Looks like `CI` is set to `true` automatically for us, fixed to use this.

https://docs.travis-ci.com/user/environment-variables/#default-environment-variables

Reply:
Looks like it didn't work :(

https://travis-ci.org/github/bitcoin-s/bitcoin-s/jobs/701135720#L765